packages/react/src/reactrouterv6.tsx

```diff
@@ -0,0 +1,177 @@
+import { Transaction, TransactionContext } from '@sentry/types';
```

Comment:
Can we make sure we attribute to whom this implementation was inspired/based on?